src/gui/dialogs/outro.cpp

```diff
@@ -31,26 +32,62 @@ namespace dialogs
 REGISTER_DIALOG(outro)
 
 outro::outro(const game_classification& info)
-	: text_(info.end_text)
+	: text_()
+	, current_text_()
 	, duration_(info.end_text_duration)
 	, fade_step_(0)
 	, fading_in_(true)
 	, timer_id_(0)
 	, next_draw_(0)
 {
-	if(text_.empty()) {
-		text_ = _("The End");
+	if(!info.end_text.empty()) {
+		text_.push_back(info.end_text);
+	} else {
+		text_.push_back(_("The End"));
 	}
 
+	text_.push_back("<span size='large'>" + info.campaign_name + "</span>");
+
+	// We only show the end text and the title if credits were turned off
+	if(info.end_credits) {
+		const auto& credits = about::get_credits_data();
+		const auto campaign_credits = std::find_if(credits.begin(), credits.end(),
+			[&info](const about::credits_group& group) { return group.id == info.campaign; });
+
+		for(const about::credits_group::about_group& about : campaign_credits->sections) {
+			// Split the names into chunks of 5
+			static const unsigned chunk_size = 5;
+			const unsigned num_names = about.names.size();
+			const unsigned num_chunks = std::max<unsigned>(1, std::ceil(num_names / chunk_size));
+
+			for(std::size_t i = 0; i < num_chunks; ++i) {
+				std::stringstream ss;
+
+				// Only include section title on first chunk
+				if(i == 0) {
+					ss << about.title << "\n";
+				}
+
+				for(std::size_t k = i * chunk_size; k < std::min<unsigned>((i + 1) * chunk_size, num_names); ++k) {
+					ss << "\n<span size='xx-small'>" << about.names[k].first << "</span>";
```

Comment:
Looks like there is a `font::escape_text()` call missing here.
```suggestion
					ss << "\n<span size='xx-small'>" << font::escape_text(about.names[k].first) << "</span>";
```

Reply:
Added this to the central handling in about.cpp.